apps-rendering/src/components/metadata.tsx

```diff
@@ -153,22 +162,22 @@ const tempraryBackgroundStyle = (format: ArticleFormat): SerializedStyles => {
 				${from.desktop} {
 					background-color: ${neutral[97]};
 				}
-				@media (prefers-color-scheme: dark) {
-					background-color: ${neutral[10]};
-				}
+
+				${darkModeCss`
+					background-color: ${background.articleContentDark(format)};
+				`}
 			`;
 		default:
 			return css`
 				background-color: ${themeStyles.liveblogBackground};
+
 				${from.desktop} {
 					background-color: ${neutral[97]};
 				}
-				@media (prefers-color-scheme: dark) {
-					background-color: ${themeStyles.liveblogDarkBackground};
-					${from.desktop} {
```

Comment:
Don't need this anymore?

Reply:
Are you double-checking this wasn't an accident?